src/pages/vebal.vue

```diff
@@ -21,7 +21,7 @@ providePoolStaking();
   <div>
     <Hero v-if="isVeBalSupported" v-once />
 
-    <div class="py-16 xl:py-20 bg-gray-50 dark:bg-gray-900">
+    <div class="py-16 xl:py-20 bg-gray-50 dark:bg-gray-850/50">
```

Comment:
Why change this?

Reply:
@pkattera

Reply:
If you look at light mode, you can see contrast vs this section and the one below. 

On dark mode, the veBAL management section and the voting table have the same background color.

Without this change, there is a really awkward bottom margin distance between the bottom of the veBAL management section and the voting table. I felt this approach using opacity was the easiest way to establish some contrast within the available color space.